src/hooks/useAppConfig.tsx

```diff
@@ -48,9 +48,10 @@ export const AppConfigProvider = ({
         return {
           ...field,
           formatValue: getFieldValueFormatter(field.fieldType),
-          isValid: (value: any) => isInputValid(value, field.rules)
+          isValid: (value: unknown) => isInputValid(value, field.rules)
         }
       })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@IlyesBenAmara can this comment be removed?

Reply:
If we remove that comment, the warning will show when we are linting the code